lib/chef/application/windows_service.rb

```diff
@@ -189,7 +189,11 @@ def run_chef_client
           config_params += " -c #{Chef::Config[:config_file]}" unless  Chef::Config[:config_file].nil?
           config_params += " -L #{Chef::Config[:log_location]}" unless Chef::Config[:log_location] == STDOUT
           # Starts a new process and waits till the process exits
-          result = shell_out("chef-client #{config_params}", :timeout => Chef::Config[:windows_service][:watchdog_timeout])
+          result = shell_out(
+            "chef-client #{config_params}",
+            :timeout => Chef::Config[:windows_service][:watchdog_timeout],
+            :logger => Chef::Log
```

Comment:
Welp. It looks like most of our use of `Chef::Log.logger` predated multiple logger support.

AFAICT, the only place we let you use the multiple logger feature (without your own tomfoolery) is when we're using a logger on STDOUT instead of a formatter, and you still want a log location. In that case, if you did `Chef::Log.logger.warn("bob")` it would only go to the default logger, which looks like it would be the logfile instead of STDOUT. So you would want `Chef::Log.warn("bob")` there. But that appears to be an edge case for people who've been around a long time and are scared of the output formatters.

So I'd leave it as `Chef::Log`. But, I would be curious if there's another use case for for multiple loggers. @danielsdeleo ?